pkg/math/voteweighted/threshold.go

```diff
@@ -0,0 +1,87 @@
+package voteweighted
+
+import (
+	"math/big"
+
+	"cosmossdk.io/math"
+)
+
+var (
+	Zero       = big.NewInt(0)
+	Two        = big.NewInt(2)
+	OneMillion = big.NewInt(1_000_000)
+)
+
+// SignsDiffer determines whether newPrice and newPrime move in different directions relative to old.
+func SignsDiffer(old *big.Int, newPrice *big.Int, newPrime *big.Int) bool {
```

Comment:
nit: can we name these oldPrice, aggregatedPrice, validatorPrice (or something similar) to indicate what values these variables correspond to intuitively

Reply:
nit: `newPrime` and `newPrice` are also a bit hard for eyes to distinguish quickly, leading to some fatigue